docs/inputs/range.md

```diff
@@ -1,3 +1,7 @@
+---
+keywords: sliders
```

Comment:
Maybe slider instead?

Reply:
It does prefix matching so this will match both.